docker-compose.yml

```diff
@@ -1,18 +1,25 @@
-version: "3.7"
+version: "3"
```

Comment:
Why downgrade to version 3?

Reply:
Well, I tought it was getting the latest one :/ but it will use `3.0` instead https://docs.docker.com/compose/compose-file/compose-versioning/#v2-and-v3-declaration

Reply:
Thanks for the explanation!